src/plugins/global-settings/colors/readme.md

```diff
@@ -0,0 +1,73 @@
+# Global Settings - Global Colors
+
+It allows users to create their own set of global color palette which
+can be applied to all blocks.
+
+## List of Action Hooks
+### ```stackable.global-settings.global-styles``` 
+Action for updating the global custom css variables in the editor.
+
+Doing this:
+
+```js
+const colors = [ 
+        { 
+            name: 'Stackable Global Color 1',
+            slug: 'stk-global-color-2532351235',
+            color: 'var(--stk-global-color-aeb2rd, #ffffff)',
+            colorVar: '--stk-global-color-aeb2rd',
+            fallback: '#ffffff',
+            rgb: '255, 255, 255',
+        } 
+    ]
+
+doAction( 'stackable-global-settings.global-styles', colors )
```

Comment:
Where is `stackable-global-settings.global-styles` used?

Reply:
It is used in the ColorPicker component.